android/build.gradle

```diff
@@ -2,7 +2,7 @@
 
 buildscript {
     ext {
-        minSdkVersion = 21
+        minSdkVersion = 30
```

Comment:
I can revert this but still wondering since our gradle version that does not support this anyways, any suggestion on what to do in this case?

Reply:
Oh I think I might be confusing concepts here since the warning is complaining about buildToolsVersion and not compileSdkVersion. This is what you mean, right?